storops/unity/resource/host.py

```diff
@@ -120,7 +196,52 @@ def get_resource_class(cls):
 
 
 class UnityHostInitiator(UnityResource):
-    pass
+    @classmethod
+    def create(cls, cli, name, host, type, is_ignored=None,
```

Comment:
Same here.  I think name is confusing, better use uid to replace it.  You actually need a wwn or iqn here, not a user specified name.